SDPi_Supplement/asciidoc/sdpi-supplement.adoc

```diff
@@ -12,6 +12,10 @@
 :imagesdir: images/
 :title-logo-image: image::ihe-logo.png[pdfwidth=4in,align=center]
 
+
+[#clause-foreword]
+= Foreword

```

Comment:
Shouldn't it be on level 2 such that the foreword is not rendered as a Book Part?
 ```asciidoc
[sdpi_offset=clear]
== Foreword
 ```

Reply:
Updated file as suggested